mail_tracking/models/mail_thread.py

```diff
@@ -35,38 +35,53 @@ def message_post(self, *args, **kwargs):
         """
         new_message = super().message_post(*args, **kwargs)
         email_cc = kwargs.get('cc')
+        toWrite = {}
         if email_cc:
-            new_message.sudo().write({
+            toWrite.update({
```

Comment:
Why not using a more compact form (both variable name and expression):
```suggestion
            vals['email_cc'] = email_cc
```